src/components/ToggleArrow/__stories__/ToggleArrow.stories.tsx

```diff
@@ -1,18 +1,69 @@
-import {Meta, StoryFn} from '@storybook/react';
+import {Meta, StoryFn, StoryObj} from '@storybook/react';
 
+import {blockTransform} from '../../../../.storybook/utils';
+import {CustomBlock} from '../../../models';
 import ToggleArrow, {ToggleArrowProps} from '../ToggleArrow';
 
 import data from './data.json';
 
 export default {
-    component: ToggleArrow,
     title: 'Components/ToggleArrow',
-} as Meta;
+    component: ToggleArrow,
+    parameters: {
+        layout: 'centered',
+        controls: {expanded: true},
+    },
+} as Meta<typeof ToggleArrow>;
 
-const DefaultTemplate: StoryFn<ToggleArrowProps> = (args) => <ToggleArrow {...args} />;
+const Template: StoryFn<ToggleArrowProps> = (args) => (
+    <ToggleArrow {...(blockTransform(args as unknown as CustomBlock) as ToggleArrowProps)} />
+);
 
-export const Horizontal = DefaultTemplate.bind({});
-export const Vertical = DefaultTemplate.bind({});
+export const Default: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const Horizontal: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const Vertical: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const OpenHorizontal: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const OpenVertical: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const Thin: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const Slow: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const ThinSlow: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const NavigationIcon: StoryObj<typeof ToggleArrow> = Template.bind({});
+export const AllVariants: StoryObj<typeof ToggleArrow> = Template.bind({});
 
+Default.args = {type: 'horizontal', iconType: 'default', size: 24};
 Horizontal.args = data.horizontal.content as ToggleArrowProps;
 Vertical.args = data.vertical.content as ToggleArrowProps;
+OpenHorizontal.args = {type: 'horizontal', open: true, size: 24};
+OpenVertical.args = {type: 'vertical', open: true, size: 24};
+Thin.args = {type: 'horizontal', thin: true, size: 24};
+Slow.args = {type: 'horizontal', slow: true, size: 24};
+ThinSlow.args = {type: 'horizontal', thin: true, slow: true, size: 24};
+NavigationIcon.args = {type: 'horizontal', iconType: 'navigation', size: 30};
+
+AllVariants.args = {};
```

Comment:
<img width="1512" height="878" alt="image" src="https://github.com/user-attachments/assets/795a1723-0038-4639-97b6-a9d90c10e968" />
controls should be array